server/models/availability.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
```

Comment:
Please adjust the base of the PR/rebase to only reflect changes pertaining to this PR